### PR TITLE
ported dev portal doc changes back

### PR DESCRIPTION
**Pull request by mDuo13**
Using [my new find/replace script](https://github.com/ripple/ripple-dev-portal/blob/gh-pages/tool/githubify.py), I commented out all the HTML blocks and replaced all the relative URLs with absolute URLs. This update contains all the changes I've made to the documentation on the dev portal since the last pull request, which is mostly typo fixes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1222,30 +1263,33 @@ The `payment` objects include additional transactional metadata:
 
 Places an order to exchange currencies.
 
+<!-- <div class='multicode'> -->
 
+*REST*
 
-```js
+```
```
Comment (lumberj) [inline README.md]: The `js` is missing here, which makes it show up nicer on github (syntax highlighting)

File: README.md [hunk 2]
```diff
@@ -569,12 +571,17 @@ Accounts are the core unit of authentication in the Ripple Network. Each account
 
 Randomly generate keys for a potential new Ripple account.
 
+<!-- <div class='multicode'> -->
+
+*REST*
```
Comment (lumberj) [inline README.md]: Wrap these in a comment block, too?

Comment (mDuo13) [inline README.md]: huh? Wrap what?

File: README.md (hunk 1 above)
Comment (mDuo13) [inline README.md]: Yeah, but in the Dev Portal, the "js" gets displayed literally when it parses the code tabs, so I intentionally don't use it on the ones that are inside the multicode areas.

File: README.md (hunk 2 above)
Comment (lumberj) [inline README.md]: *REST*

**Comment by geertweening** [comment]
Does this need further follow-up @lumberj @mDuo13

File: README.md (hunk 2 above)
Comment (lumberj) [inline README.md]: `*REST*` shows up in the README and it's just taking up space. So it would be cool to just add:
`<!-- *REST* -->`

**Comment by lumberj** [comment]
I just merged this PR into my existing one, https://github.com/ripple/ripple-rest/pull/292, since I'm already updating the `README` there

**Comment by geertweening** [comment]
So this PR should be closed and we continue our convo there?

**Comment by lumberj** [comment]
Yes. To make things easier, I'll paste the diff of the two READMEs. The main difference is that on #292 I changed `issuer` to be `counterparty` so the README reflects that. Other differences include changes to the examples of the `payment` object (`ledger`, `state` are no longer inside the object but at the root).

```diff
diff --git c/README.md w/README.md
index e015f7b..969cbab 100644
--- c/README.md
+++ w/README.md
@@ -312,7 +312,7 @@ Example error:
     "success": false,
     "error_type": "invalid_request",
     "error": "Invalid parameter: destination_amount",
-    "message": "Non-XRP payment must have an issuer"
+    "message": "Non-XRP payment must have a counterparty"
 }
 \```
 
@@ -327,7 +327,7 @@ You should parse these numbers into a numeric data type with adequate precision.
 
 There are two kinds of currency on the Ripple network: XRP, and issuances. XRP is the native cryptocurrency that only exists in the network, and can be held by accounts and sent directly to other accounts with no trust necessary.
 
-All other currencies take the form of *issuances*, which are held in the *trust lines* that link accounts. Issuances are identified by a `counterparty` on the other end of the trust line, sometimes also called the `issuer`. Sending and trading issuances actually means debiting the balance of a trust line and crediting the balance of another trust line linked to the same account. Ripple ensures this operation happens atomically.
+All other currencies take the form of *issuances*, which are held in the *trust lines* that link accounts. Issuances are identified by a `counterparty` on the other end of the trust line. Sending and trading issuances actually means debiting the balance of a trust line and crediting the balance of another trust line linked to the same account. Ripple ensures this operation happens atomically.
 
 Issuances are typically created by Gateway accounts in exchange for assets in the outside world. In most cases, the `counterparty` of a currency refers to the gateway that created the issuances. XRP never has a counterparty.
 
@@ -342,8 +342,8 @@ When an amount of currency is specified as part of a JSON body, it is encoded as
 |-------|------|-------------|
 | value | String (Quoted decimal) | The quantity of the currency |
 | currency | String | Three-digit [ISO 4217 Currency Code](http://www.xe.com/iso4217.php) specifying which currency. Alternatively, a 160-bit hex value. (Some advanced features, like [demurrage](https://ripple.com/wiki/Gateway_demurrage), require the hex version.) |
-| counterparty | String | (New in [v1.4.0](https://github.com/ripple/ripple-rest/releases/tag/1.4.0-rc1)) The Ripple address of the account that is a counterparty to this currency. This is usually an [issuing gateway](https://wiki.ripple.com/Gateway_List). Always omitted, or an empty string, for XRP. |
-| issuer | String | (Prior to 1.4.0) **DEPRECATED** alias for `counterparty`. Some methods may still return this instead. |
+| counterparty | String | (New in [v1.3.2](https://github.com/ripple/ripple-rest/releases/tag/1.3.2-rc4)) The Ripple address of the account that is a counterparty to this currency. This is usually an [issuing gateway](https://wiki.ripple.com/Gateway_List). Always omitted, or an empty string, for XRP. |
+| issuer | String | (Prior to 1.4.0) **DEPRECATED** alias for `counterparty`. |
 
 
 Example Amount Object:
@@ -407,7 +407,7 @@ An example Payment object looks like this:
     "source_amount": {
         "value": "0.001",
         "currency": "XRP",
-        "issuer": ""
+        "counterparty": ""
     },
     "source_slippage": "0",
     "destination_address": "rNw4ozCG514KEjPs5cDrqEcdsi31Jtfm5r",
@@ -415,7 +415,7 @@ An example Payment object looks like this:
     "destination_amount": {
         "value": "0.001",
         "currency": "XRP",
-        "issuer": ""
+        "counterparty": ""
     },
     "invoice_id": "",
     "paths": "[]",
@@ -448,14 +448,14 @@ Submitted transactions can have additional fields reflecting the current status
 | Field | Type | Description |
 |-------|------|-------------|
 | direction | String | The direction of the payment relative to the account from the URL, either `"outgoing"` (sent by the account in the URL) or `"incoming"` (received by the account in the URL) |
-| state | String | The current status of the payment in transaction processing. A value of `"validated"` indicates that the transaction is finalized. |
 | result | String | The [Ripple transaction status code](https://wiki.ripple.com/Transaction_errors) for the transaction. A value of `"tesSUCCESS"` indicates a successful transaction. |
-| ledger | String | The sequence number of the ledger version that includes this transaction. |
-| hash | String | A hash value that uniquely identifies this transaction in the Ripple network. |
 | timestamp | String | The time the ledger containing this transaction was validated, as a [ISO8601 extended format](http://en.wikipedia.org/wiki/ISO_8601) string in the form `YYYY-MM-DDTHH:mm:ss.sssZ`. |
 | fee | String (Quoted decimal) | The amount of XRP charged as a transaction fee. |
+| balance_changes | Array | Array of [Amount objects](#amount_object) indicating changes in balances held by the perspective account (i.e., the Ripple account address specified in the URI). |
 | source_balance_changes | Array | Array of [Amount objects](#amount_object) indicating changes in balances held by the account sending the transaction as a result of the transaction. |
 | destination_balance_changes | Array | Array of [Amount objects](#amount_object) indicating changes in balances held by the account receiving the transaction as a result of the transaction. |
+| destination_amount_submitted | Object | An [Amount object](#amount_object) indicating the destination amount submitted (useful when `payment.partial_payment` flag is set to *true* |
+| source_amount_submitted | Object | An [Amount object](#amount_object) indicating the source amount submitted (useful when `payment.partial_payment` flag is set to *true* |
 
 
 ### Memo Objects ###
@@ -573,7 +573,7 @@ Randomly generate keys for a potential new Ripple account.
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 GET /v1/wallet/new
@@ -614,7 +614,7 @@ Retrieve the current balances for the given Ripple account.
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 GET /v1/accounts/{:address}/balances
@@ -682,7 +682,7 @@ Retrieve the current settings for a given account.
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 GET /v1/accounts/{:address}/settings
@@ -749,7 +749,7 @@ Modify the existing settings for an account.
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 POST /v1/accounts/{:address}/settings?validated=true
@@ -848,7 +848,7 @@ Get quotes for possible ways to make a particular payment.
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 GET /v1/accounts/{:source_address}/payments/paths/{:destination_address}/{:amount}
@@ -888,7 +888,7 @@ You can then choose one of the returned payment objects, modify it as desired (f
       "source_amount": {
         "value": "1.008413509923106",
         "currency": "USD",
-        "issuer": ""
+        "counterparty": ""
       },
       "source_slippage": "0",
       "destination_account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
@@ -896,7 +896,7 @@ You can then choose one of the returned payment objects, modify it as desired (f
       "destination_amount": {
         "value": "1",
         "currency": "USD",
-        "issuer": "rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q"
+        "counterparty": "rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q"
       },
       "invoice_id": "",
       "paths": "[[{\"account\":\"rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B\",\"type\":1,\"type_hex\":\"0000000000000001\"},{\"currency\":\"USD\",\"issuer\":\"rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q\",\"type\":48,\"type_hex\":\"0000000000000030\"},{\"account\":\"rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q\",\"type\":1,\"type_hex\":\"0000000000000001\"}],[{\"account\":\"rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B\",\"type\":1,\"type_hex\":\"0000000000000001\"},{\"currency\":\"XRP\",\"type\":16,\"type_hex\":\"0000000000000010\"},{\"currency\":\"USD\",\"issuer\":\"rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q\",\"type\":48,\"type_hex\":\"0000000000000030\"},{\"account\":\"rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q\",\"type\":1,\"type_hex\":\"0000000000000001\"}]]",
@@ -909,7 +909,7 @@ You can then choose one of the returned payment objects, modify it as desired (f
       "source_amount": {
         "value": "61.06103",
         "currency": "XRP",
-        "issuer": ""
+        "counterparty": ""
       },
       "source_slippage": "0",
       "destination_account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
@@ -917,7 +917,7 @@ You can then choose one of the returned payment objects, modify it as desired (f
       "destination_amount": {
         "value": "1",
         "currency": "USD",
-        "issuer": "rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q"
+        "counterparty": "rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q"
       },
       "invoice_id": "",
       "paths": "[[{\"currency\":\"USD\",\"issuer\":\"rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q\",\"type\":48,\"type_hex\":\"0000000000000030\"},{\"account\":\"rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q\",\"type\":1,\"type_hex\":\"0000000000000001\"}],[{\"currency\":\"USD\",\"issuer\":\"rpDMez6pm6dBve2TJsmDpv7Yae6V5Pyvy2\",\"type\":48,\"type_hex\":\"0000000000000030\"},{\"account\":\"rpDMez6pm6dBve2TJsmDpv7Yae6V5Pyvy2\",\"type\":1,\"type_hex\":\"0000000000000001\"},{\"account\":\"rHAwwozJw6FHfnJfRQaFXrkGHocGoaNYSy\",\"type\":1,\"type_hex\":\"0000000000000001\"},{\"account\":\"rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q\",\"type\":1,\"type_hex\":\"0000000000000001\"}],[{\"currency\":\"USD\",\"issuer\":\"rsP3mgGb2tcYUrxiLFiHJiQXhsziegtwBc\",\"type\":48,\"type_hex\":\"0000000000000030\"},{\"account\":\"rsP3mgGb2tcYUrxiLFiHJiQXhsziegtwBc\",\"type\":1,\"type_hex\":\"0000000000000001\"},{\"account\":\"rEtr3Kzh5MmhPbeNu6PDtQZsKBpgFEEEo5\",\"type\":1,\"type_hex\":\"0000000000000001\"},{\"account\":\"rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q\",\"type\":1,\"type_hex\":\"0000000000000001\"}],[{\"currency\":\"USD\",\"issuer\":\"rsP3mgGb2tcYUrxiLFiHJiQXhsziegtwBc\",\"type\":48,\"type_hex\":\"0000000000000030\"},{\"account\":\"rsP3mgGb2tcYUrxiLFiHJiQXhsziegtwBc\",\"type\":1,\"type_hex\":\"0000000000000001\"},{\"account\":\"rKvPTQrD8ap1Y8TSmKjcK6G7q7Kvx7RAqQ\",\"type\":1,\"type_hex\":\"0000000000000001\"},{\"account\":\"rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q\",\"type\":1,\"type_hex\":\"0000000000000001\"}]]",
@@ -941,7 +941,7 @@ Submit a payment object to be processed and executed.
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 POST /v1/accounts/{address}/payments?validated=true
@@ -958,7 +958,7 @@ POST /v1/accounts/{address}/payments?validated=true
     "source_amount": {
       "value": "5.01",
       "currency": "USD",
-      "issuer": ""
+      "counterparty": ""
     },
     "source_slippage": "0",
     "destination_account": "r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59",
@@ -966,7 +966,7 @@ POST /v1/accounts/{address}/payments?validated=true
     "destination_amount": {
       "value": "5",
       "currency": "USD",
-      "issuer": ""
+      "counterparty": ""
     },
     "invoice_id": "",
     "paths": "[[{\"account\":\"rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B\",\"type\":1,\"type_hex\":\"0000000000000001\"}]]",
@@ -1039,7 +1039,7 @@ Retrieve the details of a payment, including the current state of the transactio
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 GET /v1/accounts/{:address}/payments/{:id}
@@ -1067,50 +1067,65 @@ The following URL parameters are required by this API endpoint:
     "source_amount": {
       "value": "0.00001",
       "currency": "XRP",
-      "issuer": ""
+      "counterparty": ""
     },
     "source_slippage": "0",
     "destination_account": "ra5nK24KXen9AHvsdFTKHSANinZseWnPcX",
     "destination_tag": "",
     "destination_amount": {
+      "value": "0.00000005080000000000001",
       "currency": "USD",
-      "issuer": "rsP3mgGb2tcYUrxiLFiHJiQXhsziegtwBc",
-      "value": "0.01"
+      "counterparty": "rsP3mgGb2tcYUrxiLFiHJiQXhsziegtwBc"
     },
     "invoice_id": "",
     "paths": "[]",
     "no_direct_ripple": false,
     "partial_payment": true,
     "direction": "outgoing",
-    "state": "validated",
     "result": "tesSUCCESS",
-    "ledger": "8924146",
-    "hash": "9D591B18EDDD34F0B6CF4223A2940AEA2C3CC778925BABF289E0011CD8FA056E",
     "timestamp": "2014-09-17T21:47:00.000Z",
     "fee": "0.00001",
-    "source_balance_changes": [
-      {
-        "value": "-0.00002",
-        "currency": "XRP",
-        "issuer": ""
-      }
-    ],
-    "destination_balance_changes": [
-      {
-        "value": "5.08e-8",
-        "currency": "USD",
-        "issuer": "rsP3mgGb2tcYUrxiLFiHJiQXhsziegtwBc"
-      }
-    ]
-  }
+    "balance_changes": [{
+      "currency": "XRP",
+      "value": "-0.00002",
+      "counterparty": ""
+    }],
+    "source_balance_changes": [{
+      "currency": "XRP",
+      "value": "-0.00002",
+      "counterparty": ""
+    }],
+    "destination_balance_changes": [{
+      "currency": "USD",
+      "value": "5.08e-8",
+      "counterparty": "rsP3mgGb2tcYUrxiLFiHJiQXhsziegtwBc"
+    }],
+    "order_changes": [],
+    "destination_amount_submitted": {
+      "value": "0.01",
+      "currency": "USD",
+      "counterparty": "rsP3mgGb2tcYUrxiLFiHJiQXhsziegtwBc"
+    },
+    "source_amount_submitted": {
+      "value": "0.00001",
+      "currency": "XRP",
+      "counterparty": ""
+    }
+  },
+  "client_resource_id": "",
+  "hash": "9D591B18EDDD34F0B6CF4223A2940AEA2C3CC778925BABF289E0011CD8FA056E",
+  "ledger": "8924146",
+  "state": "validated"
 }
 \```
 
 | Field | Type | Description |
 |-------|------|-------------|
 | payment | Object | A [payment object](#payment-objects) for the transaction. |
+| hash | String | The hash of the payment transaction |
+| ledger | String | The sequence number of the ledger version that includes this transaction. |
 
-If the `payment.state` field has the value `"validated"`, then the payment has been finalized, and is included in the shared global ledger. However, this does not necessarily mean that it succeeded. Check the `payment.result` field for a value of `"tesSUCCESS"` to see if the payment was successfully executed. If the `payment.partial_payment` flag is *true*, then you should also consult the `payment.destination_balance_changes` array to see how much currency was actually delivered to the destination account.
+If the `state` field has the value `"validated"`, then the payment has been finalized, and is included in the shared global ledger. However, this does not necessarily mean that it succeeded. Check the `payment.result` field for a value of `"tesSUCCESS"` to see if the payment was successfully executed. If the `payment.partial_payment` flag is *true*, then you should also consult the `payment.destination_balance_changes` array to see how much currency was actually delivered to the destination account.
 
 Processing a payment can take several seconds to complete, depending on the [consensus process](https://ripple.com/consensus-whitepaper/). If the payment does not exist yet, or has not been validated, you should wait a few seconds before checking again.
 
@@ -1123,7 +1138,7 @@ Retrieve a selection of payments that affected the specified account.
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 GET /v1/accounts/{:address}/payments
@@ -1158,100 +1173,58 @@ Optionally, you can also include the following query parameters:
 \```js
 {
   "success": true,
-  "payments": [
-    {
-      "payment": {
-        "source_account": "rBvktWhzs4MQDaFYScsqPCB5YufRDXwKDC",
-        "source_tag": "",
-        "source_amount": {
-          "value": "1166313.057",
-          "currency": "JPY",
-          "issuer": "r94s8px6kSw1uZ1MV98dhSRTvc6VMPoPcN"
-        },
-        "source_slippage": "0",
-        "destination_account": "rBvktWhzs4MQDaFYScsqPCB5YufRDXwKDC",
-        "destination_tag": "",
-        "destination_amount": {
-          "value": "600000",
-          "currency": "XRP",
-          "issuer": ""
-        },
-        "invoice_id": "",
-        "paths": "[[{\"currency\":\"XRP\",\"type\":16,\"type_hex\":\"0000000000000010\"}]]",
-        "no_direct_ripple": false,
-        "partial_payment": false,
-        "direction": "passthrough",
-        "result": "tesSUCCESS",
-        "timestamp": "2015-02-06T03:59:30.000Z",
-        "fee": "0.012",
-        "source_balance_changes": [
-          {
-            "currency": "JPY",
-            "value": "-1164533.852420654",
-            "issuer": "r94s8px6kSw1uZ1MV98dhSRTvc6VMPoPcN"
-          },
-          {
-            "currency": "XRP",
-            "value": "599999.988",
-            "issuer": ""
-          }
-        ],
-        "destination_balance_changes": [
-          {
-            "currency": "JPY",
-            "value": "-1164533.852420654",
-            "issuer": "r94s8px6kSw1uZ1MV98dhSRTvc6VMPoPcN"
-          },
-          {
-            "currency": "XRP",
-            "value": "599999.988",
-            "issuer": ""
-          }
-        ],
-        "order_changes": [
-          {
-            "taker_pays": {
-              "currency": "JPY",
-              "counterparty": "r94s8px6kSw1uZ1MV98dhSRTvc6VMPoPcN",
-              "value": "-67605.3"
-            },
-            "taker_gets": {
-              "currency": "XRP",
-              "counterparty": "",
-              "value": "-34300"
-            },
-            "sequence": 3751,
-            "status": "closed"
-          }
-        ],
-        "memos": [
-          {
-            "MemoType": "636C69656E74",
-            "MemoFormat": "7274312E332E31",
-            "parsed_memo_type": "client",
-            "parsed_memo_format": "rt1.3.1"
-          }
-        ]
+  "payments": [{
+    "payment": {
+      "source_account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+      "source_tag": "",
+      "source_amount": {
+        "value": "20",
+        "currency": "XRP",
+        "counterparty": ""
       },
-      "client_resource_id": "",
-      "hash": "E485D1E18D946ACD410AD79F51E2C57E887CC206286E6CE0A1CA80FC75C24643",
-      "ledger": "11547185",
-      "state": "validated"
-    }
-  ]
-}
+      "source_slippage": "0",
+      "destination_account": "raKEEVSGnKSD9Zyvxu4z6Pqpm4ABH8FS6n",
+      "destination_tag": "",
+      "destination_amount": {
+        "value": "20",
+        "currency": "XRP",
+        "counterparty": ""
+      },
+      "invoice_id": "",
+      "paths": "[]",
+      "no_direct_ripple": false,
+      "partial_payment": false,
+      "direction": "outgoing",
+      "result": "tesSUCCESS",
+      "timestamp": "2015-02-09T23:31:40.000Z",
+      "fee": "0.012",
+      "balance_changes": [{
+        "currency": "XRP",
+        "value": "-20.012",
+        "counterparty": ""
+      }],
+      "source_balance_changes": [{
+        "currency": "XRP",
+        "value": "-20.012",
+        "counterparty": ""
+      }],
+      "destination_balance_changes": [{
+        "currency": "XRP",
+        "value": "20",
+        "counterparty": ""
+      }],
+      "order_changes": []
+    },
+    "client_resource_id": "",
+    "hash": "1D381C0FCA00E8C34A6D4D3A91DAC9F3697B4E66BC49ED3D9B2D6F57D7F15E2E",
+    "ledger": "11620700",
+    "state": "validated"
+  }
+]
 \```
 
 If the length of the `payments` array is equal to `results_per_page`, then there may be more results. To get them, increment the `page` query paramter and run the request again.
 
-The `payment` objects include additional transactional metadata:
-
-| Field | Type | Description |
-|-------|------|-------------|
-| source_balance_changes | Amount | The balance changes of the Ripple address that submitted the payment |
-| destination_balance_changes | Amount | The balance changes of the Ripple address that received the payment |
-| order_changes | Order | The changes in the orders of the perspective account |
-
 *Note:* It is not more efficient to specify more filter values, because Ripple-REST has to retrieve the full list of payments from the `rippled` before it can filter them.
 
 # ORDERS #
@@ -1265,7 +1238,7 @@ Places an order to exchange currencies.
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 POST /v1/accounts/{:address}/orders?validated=true
@@ -1346,7 +1319,7 @@ Deletes a previous order to exchange currencies.
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 DELETE /v1/accounts/{:address}/orders/{:order}?validated=true
@@ -1406,7 +1379,7 @@ Retrieves all open currency-exchange orders associated with the Ripple address.
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 GET /v1/accounts/{:address}/orders
@@ -1605,7 +1578,7 @@ Get the details of an order transaction. An order transaction either [places an
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 GET /v1/accounts/{:address}/orders/{:hash}
@@ -1738,7 +1711,7 @@ Retrieves the top of the order book for a currency pair.
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 GET /v1/accounts/{:address}/order_book/{:base}/{:counter}
@@ -1911,7 +1884,7 @@ Retrieves all trustlines associated with the Ripple address.
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 GET /v1/accounts/{:address}/trustlines
@@ -1982,7 +1955,7 @@ Creates or modifies a trustline.
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 POST /v1/accounts/{:address}/trustlines?validated=true
@@ -2056,7 +2029,7 @@ Get a notification for the specific transaction hash, along with links to previo
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 GET /v1/accounts/{:address}/notifications/{:id}
@@ -2124,7 +2097,7 @@ Perform a simple ping to make sure that the server is working properly.
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 GET /v1/server/connected
@@ -2152,7 +2125,7 @@ Retrieve information about the current status of the Ripple-REST API and the `ri
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 GET /v1/server
@@ -2241,7 +2214,7 @@ Returns a Ripple transaction, in its complete, original format.
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 GET /v1/transactions/{:id}
@@ -2448,7 +2421,7 @@ Retrieve the current transaction fee, in XRP, for the `rippled` server Ripple-RE
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 GET /v1/transaction-fee
@@ -2476,7 +2449,7 @@ Generate a universally-unique identifier suitable for use as the Client Resource
 
 <!-- <div class='multicode'> -->
 
-*REST*
+<!-- *REST* -->
 
 \```
 GET /v1/uuid
```

**Pull request closed without merging**